input/index.cshtml

```diff
@@ -3,6 +3,28 @@ NoSidebar: true
 NoContainer: true
 NoGutter: true
 ---
+@{
+    // Latest Cake release name from GitHub e.g. v1.0.0-rc0002
+    var latestReleaseName = Model.String("CakeLatestReleaseName") ?? "x.y.z";
```

Comment:
`x.y.z` doesn't seem like a useful fallback. Wouldn't just hide the banner be a better option if no information is passed?

Reply:
The `x.y.z` should never really be hit unless you comment something out in `build.cake` / `github.cake` during local development, so it's just a local dev fallback to see something in the banner.